docs/guides/hedgehog-engine/rangers/Cyloop/color.md

```diff
@@ -0,0 +1,8 @@
+---
+description: Cyloop color replacement guide for Sonic Frontiers
+---
+# Replacing Cyloop Color
+
+!!! info
+
+    - [Sonic Frontiers Cyloop Color Doc](https://docs.google.com/document/d/1OyAfEtlemIAyebN8KUwIg7ZSvgJuy06BAewPoFWrJEs/edit?usp=sharing){ target="_blank"}
```

Comment:
The point of HedgeDocs is to have the information stored on the website itself, and not link to other external websites. Please copy the guide's content to this markdown file, and follow the guidelines/the style of other guides present in HedgeDocs already.